### PR TITLE
Fixed doc compilation warnings

### PR DESCRIPTION
**Pull request by gastaldi**

### CODE REVIEW

File: docs/src/main/asciidoc/amazon-lambda.adoc [hunk 1]
```diff
@@ -480,7 +480,7 @@ when using synchronous mode, due to issues in the GraalVM compilation (at presen
 Add `quarkus-jaxb` as a dependency in your Maven `pom.xml`, or Gradle `build.gradle` file.
 
 You must also force your AWS service client for SQS, SNS, S3 et al, to use the URL Connection client,
-which connects to AWS services over HTTPS, hence the inclusion of the SSL enabled property, as described in the <<Using HTTPS>> section above.
+which connects to AWS services over HTTPS, hence the inclusion of the SSL enabled property, as described in the <<Using HTTPS or SSL/TLS>> section above.
```
Comment (gsmet) [inline docs/src/main/asciidoc/amazon-lambda.adoc]: We need to use an id here to avoid that kind of issues.

**Review by gsmet — CHANGES_REQUESTED** [review]
Good catch, comment inline.

File: docs/src/main/asciidoc/amazon-lambda.adoc (hunk 1 above)
Comment (gastaldi) [reply docs/src/main/asciidoc/amazon-lambda.adoc]: done

**Review by gsmet — APPROVED** [review]

**Comment by gsmet** [comment]
Ok, I will have to backport only part of it as I'm not sure the lambda stuff is in the 1.3 branch.

Will let Git do the work for me :).

**Comment by gsmet** [comment]
Apparently, these issues were only in master.

**Pull request merged**